lib/Sema/BuilderTransform.cpp

```diff
@@ -1202,10 +1202,14 @@ class BuilderClosureRewriter
     ifStmt->setThenStmt(newThen);
 
     // Look for a #available condition. If there is one, we need to check
-    // that the resulting type of the "then" does refer to any types that
+    // that the resulting type of the "then" doesn't refer to any types that
     // are unavailable in the enclosing context.
     //
-    // Note that this is for staging in support for
+    // Note that this is for staging in support for buildLimitedAvailability();
+    // the diagnostic is currently a warning, so that existing code that
+    // compiles today will continue to compile. Once function builder types
+    // have had the change to adopt buildLimitedAvailability(), we'll upgrade
```

Comment:
Did you mean "chance" instead of "change"? :-)
```suggestion
    // have had the chance to adopt buildLimitedAvailability(), we'll upgrade
```

Reply:
ARGH! I did.